pupil_src/shared_modules/marker_tracker_3d/localization.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+
+from marker_tracker_3d import utils
+
+
+class Localization:
+    def __init__(self, camera_model, marker_model):
+        self.camera_model = camera_model
+        self.marker_model = marker_model
```

Comment:
`marker_model` seems to live both in `storage` and here.

Reply:
My thought was: only controllers (`marker_detector`, `optimization_controller`, `camera_localizer`) have access to  `storage`, and the needed arguments are passed to the `Localization` and `Optimization`.

Reply:
According to the Model-View-Controller dogma everything that implements program logic belongs to the controller part of the source code. With that definition `Optimization` and `Localization` are also controllers.
I don't see a reason to treat `Optimization` and `Localization` differently here. If we have two versions of `marker_model` there is a risk of them falling out of sync.